roles/edpm_libvirt/defaults/main.yml

```diff
@@ -62,7 +62,11 @@ edpm_libvirt_packages:
   # ceph-common is needed for ceph support to provide the ceph client lib
   # for qemu and the ceph udev rules
   - ceph-common
+  # for SASL auth
+  - cyrus-sasl-scram
 edpm_libvirt_ceph_path: /var/lib/openstack/config/ceph
+edpm_libvirt_password_path: /var/lib/openstack/configs/{{ edpm_service_name | default('libvirt') }}/LibvirtPassword
+edpm_libvirt_sasl_auth_enabled: "{{ edpm_libvirt_password_path is exists | bool }}"
```

Comment:
i do nnot think this shoul dbe configurable separate form the generic tls support

we do not want to support sasl auth without tls as we do not want to support connecting to libvirt vai the network without tls.

so we should just enable sasl auth if edpm_libvirt_tls_certs_enabled is true

Reply:
you need the secret too

Reply:
> we do not want to support sasl auth without tls as we do not want to support connecting to libvirt vai the network without tls.

Can't do that. With this can have TLS, or TLS+SASL auth, nothing else

Reply:
... but doesn't seem to be optional on tripleo so no reason to disable it